examples/README.md

```
@@ -138,6 +140,33 @@ def serve():
     ...
 ```
 
+### Progress Monitoring
+
+By default, `client.deploy()` and `deployment.wait_until_ready()` show progress output:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)